ui/src/app/utils/access-utils.tsx

```diff
@@ -451,6 +460,10 @@ export const computeRenewalDisplayDates = ({
         return {
           lastConfirmedDate,
           nextReviewDate: `${nextReviewDate} ${daysRemainingDisplay}`,
+          moduleStatus:
+            daysRemaining >= 0
```

Comment:
I don't think this is quite enough.  We also want to express whether a module is **expiring soon**